pkg/api/service/annotations.go

```diff
@@ -25,4 +25,9 @@ const (
 	//
 	// Not all cloud providers support this annotation, though AWS & GCE do.
 	AnnotationLoadBalancerSourceRangesKey = "service.beta.kubernetes.io/load-balancer-source-ranges"
+
+	// An annotation that denotes if this Service desires to only route to local
+	// endpoints. Typically used to preserve source IP in loadbalanced Services.
+	AnnotationOnlyNodeLocalEndpoints = "service.alpha.kubernetes.io/cloud-lb-only-node-local-endpoints"
+	AnnotationHealthCheckNodePort    = "service.alpha.kubernetes.io/healthcheck-nodeport-node-local-endpoints"
```

Comment:
isn't this health check annotation usable without nodelocal endpoints? meaning if i simply want to scrape number of endpoints per node, I'd set it to some random unused port and kube-proxy would reply on that port with len(endpoints)?